src/command_modules/azure-cli-acs/setup.py

```diff
@@ -14,7 +14,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "2.3.13"
+VERSION = "2.3.14"
```

Comment:
Same comment as above--I think we can leave it at 2.3.13 if it's going into the current sprint.